hw/ip/dma/data/dma_testplan.hjson

```diff
@@ -210,6 +210,14 @@
         stage: V2
         tests: ["dma_memory_stress"]
       }
+      {
+        name: dma_generic_stress
+        desc: '''
+              Stress test in which random DMA operations are performed with 'memory-to-memory' and 'hardware handshaking' mode.
```

Comment:
@alees24: You may be able to provide a more detailed description of this test point?